### ISSUE TITLE
bug: "Returns:" section is not shown

### ISSUE THREAD

**Issue by shunichironomura** [mkdocstrings/griffe-typingdoc#9]
### Description of the bug
The "Returns:" section is not displayed for a function with the return value annotated.

### To Reproduce

Run the following commands:
```
git clone https://github.com/mkdocstrings/griffe-typingdoc.git
cd griffe-typingdoc/
make setup
```

Append the following lines to the `docs/examples/simple.py`:

```python
def return_value(
    param: Annotated[int, Doc("Parameter")],
) -> Annotated[int, Doc("Returned value.")]:
    """Showing off return values."""
```

Then serve the documentation:

```
make docs
```

### Full traceback

Not applicable. See the additional context section for the screenshot of the generated documentation.

### Expected behavior

In the [documentation section for `return_value` function](http://127.0.0.1:8000/griffe-typingdoc/examples/#simple.return_value), the "Returns:" section is displayed with the description of the return value "Returned value."

### Environment information

```bash
python -m griffe_typingdoc.debug  # | xclip -selection clipboard
```

- __System__: Linux-5.15.133.1-microsoft-standard-WSL2-x86_64-with-glibc2.35
- __Python__: cpython 3.12.1
- __Environment variables__:
- __Installed packages__:
  - `griffe-typingdoc` v0.2.4+editable

### Additional context
Screenshot of the generated documentation:

![image](https://github.com/mkdocstrings/griffe-typingdoc/assets/10000804/c0621190-59ae-424f-a87a-acd6c1c0ff9b)

**Comment by pawamoy** [comment]
Thanks for the report, I was able to confirm. Fixing ASAP!

**Comment by pawamoy** [comment]
Fixed in v0.2.5 :slightly_smiling_face:
